io/DerivativeWriter.cpp

```diff
@@ -161,9 +160,17 @@ void DerivativeWriter::write(const PointViewPtr data)
                     continue;
                 Matrix3d block = paddedDSM.block(r-1, c-1, 3, 3);
                 if (to.m_type == SLOPE_D8)
+                {
```

Comment:
I'll push a comment. Couldn't find the predefined PI though. Maybe I'm missing something. We could use this in a handful of places, if it indeed exists.